tests/foreman/api/test_registration.py

```diff
@@ -88,9 +87,10 @@ def test_host_registration_end_to_end(
     assert CLIENT_PORT == rhel_contenthost.subscription_config['server']['port']
 
 
+@pytest.mark.no_containers
```

Comment:
Could you please justify the usage of a VM? There is a significant test time increase if we use VMs.
Why is container not a good fit for this testcase?

Reply:
This was not intended. I added it for testing something else. Removed it.